src/stores/ConfigStore.ts

```diff
@@ -49,6 +52,13 @@ export const useConfigStore = createStore<ConfigState>('ConfigStore', (_, get) =
   },
   accessModel: 'SVOD',
   adScheduleData: null,
+  apiHost: import.meta.env.APP_API_BASE_URL || API_HOST.prd,
+  getPlayer: (): string => {
+    const host = get().apiHost;
+
+    // For dev api host we use only one testing id, for prod it can be also set in .env file
+    return host === API_HOST.dev ? DEFAULT_PLAYER.dev : import.meta.env.APP_DEFAULT_PLAYER ?? DEFAULT_PLAYER.prd;
```

Comment:
It's a little non-intuitive that this check depends on comparing the host to the host constant and also doing this as a method instead of just checking and setting it once. I suggest doing this check in `loadAndValidateConfig` in configLoad.ts.